ts/output/chtml/Wrappers/mrow.ts

```diff
@@ -100,15 +100,27 @@ export const ChtmlMrow = (function <N, T, D>(): ChtmlMrowClass<N, T, D> {
         display: 'inline-table',
         width: '100%'
       },
-      'mjx-linestack[data-mjx-breakable]': {
-        display: 'inline',
-        width: 'initial',
+      'mjx-linestack[breakable]': {
+        display: 'inline'
       },
-      'mjx-linestack[data-mjx-breakable] > mjx-linebox': {
+      'mjx-linestack[breakable] > mjx-linebox': {
         display: 'inline'
       },
-      'mjx-break[newline]::after': {
-        display: 'block'
+      'mjx-linestack[breakable] > mjx-linebox::before': {
+        'white-space': 'pre',
+        content: '"\\A"'
```

Comment:
Why do we need this `content`?

Reply:
For inline math that contains explicit line breaks (via `\\` in TeX or `<mo linebreak="newline">` in MathML), the linebreaking algorithm will insert `mjx-linestack` and `mix-linebox` elements.  The latter are normally `display: block`, which make the line break, but for in-line math, the `mjx-linebox` nodes are `display: inline` and on't force a line brake.  This `content` and `white-space` is what forces the linebreak at the end of the `mjx-linebox` in in-line math.